frontend/src/components/interview/interview.tsx

```diff
@@ -1,10 +1,16 @@
-import { DataStatus, PaginationDefaultValue } from 'common/enums/enums';
+import {
+  DataStatus,
+  PaginationDefaultValue,
+  PermissionKey,
+} from 'common/enums/enums';
 import {
   FC,
   InterviewsGetAllItemResponseDto,
   InterviewsUpdateRequestDto,
+  UserWithPermissions,
 } from 'common/types/types';
 import { Spinner } from 'components/common/common';
+import { checkHasPermission } from 'guruhub-shared';
```

Comment:
Import in components should not be directly from shared folder. Firstly import this helper in helpers folder. As far as I remember, this helper is already there, so just imort it from helpers folder in frontend

Reply:
yep, you're right